### PR TITLE
add dilation case for ConvTranspose op

### PR DESCRIPTION
**Pull request by RandyShuai**
Add case to test ConvTranspose op with dilation.

### CODE REVIEW

**Comment by raymondxyang** [comment]
please also push the updated doc, and generated test data; see https://github.com/onnx/onnx/blob/master/docs/AddNewOp.md#step-5-update-the-doc-and-generate-the-test-data for help

**Comment by RandyShuai** [comment]
new test model uploaded.

**Comment by raymondxyang** [comment]
@houseroad Hi lu just want to confirm the addition of testcase wont break things on ur side

File: docs/Operators.md [hunk 1]
```diff
@@ -2687,6 +2687,30 @@ expect(node, inputs=[x, W], outputs=[y],
...
+```python
+x = np.array([[[[3., 8., 1.],  # (1, 1, 3, 3)
+                [9., 5., 7.],
+                [3., 2., 6.]]]]).astype(np.float32)
+W = np.array([[[[7., 2.],  # (1, 1, 2, 2)
+                [1., 9.]]]]).astype(np.float32)
+
+node = onnx.helper.make_node("ConvTranspose", ["X", "W"], ["Y"], dilations=[2, 2])
+
+y = np.array([[[[21., 56., 13., 16., 2.],  # [1, 1, 5, 5]
```
Comment (houseroad) [inline docs/Operators.md]: I checked, the results I got was:

tensor([[[[21.1887, 56.1887, 13.1887, 16.1887,  2.1887],
          [63.1887, 35.1887, 67.1887, 10.1887, 14.1887],
          [24.1887, 22.1887, 76.1887, 76.1887, 21.1887],
          [ 9.1887,  5.1887, 88.1887, 45.1887, 63.1887],
          [ 3.1887,  2.1887, 33.1887, 18.1887, 54.1887]]]]

Did I miss something here?

Comment (RandyShuai) [reply docs/Operators.md]: Mind to share your script here?

Comment (houseroad) [reply docs/Operators.md]: Never mind, I forgot to set bias. Apparently, that noise is from bias.

The result look good to me now.

**Pull request merged**